### DISCUSSION TITLE
watching a URL or adding a provider?

### DISCUSSION THREAD

**Discussion by ko56** [Goldenfreddy0703/Otaku#356] [Ideas]
Thanks for making Otaku available.
Is there a way to watch a particular video, specified by a URL? For example,
[https://anime4i.com/watch-the-westward-season-5/ep-30-sv1.html](https://github.com/Goldenfreddy0703/Otaku/discussions/url)?
If not, is it possible to add anime4i.com as a provider, or something like that?
Thanks.

**Comment by Goldenfreddy0703** [comment]
Sadly no, i think i explained it in another discussion. We do have a watch history if that helps.
